assets/airports/kbos.json

```diff
@@ -596,6 +596,56 @@
         ]
       },
       {
+        "category": "departure",
+        "origin": "kbos",
+        "destination": null,
+        "method": "random",
+        "route": "KBOS.CELTK4.CELTK",
+        "rate": 5,
+        "altitude": [32000, 40000],
+        "speed": null,
```

Comment:
It is *highly* advisable that a speed be set. If everybody is allowed to fly at their cruise speed (which would presumably be the resulting behavior), the spacing we spawn with will surely deteriorate by the time they arrive at the airspace boundary. I'd recommend as part of the validation, if any, that a speed be required. I personally tend to use 320 if all jets, or 280 if includes turboprops, just to ensure they can all travel at the same speed.